packages/data-sdk/package.json

```diff
@@ -28,7 +28,7 @@
   "scripts": {
     "preversion": "npm run verify",
     "postversion": "make",
-    "dev": "vite --port 5173",
+    "dev": "vite --port 5173 --host 0.0.0.0",
```

Comment:
Why opening up to all hosts? Could be an issue if working at coffee shops, random wifi, etc...

Reply:
Working on VMs / remote hosts etc. requires this. I plan to remove this actually, and anyone using alternative dev paths can add :`npm run dev --host 0.0.0.0`

Reply:
many parts of this repo already have this:

<img width="513" alt="image" src="https://github.com/FormantIO/toolkit/assets/35079898/1d68601a-c473-4af9-875e-953eeeef7076">